{{ cookiecutter.repo_name }}/docs/aqa_frameworks/GDS/assumptions_caveats.md

```diff
@@ -0,0 +1,58 @@
+# Assumptions and caveats Log
+
+This log contains a list of assumptions and caveats used in this analysis.
+
+<!-- Use reStructuredText contents directive to generate a local contents -->
+```eval_rst
+.. contents::
+    :local:
+    :depth: 1
+```
+
+## Definitions
+
+Assumptions are RAG-rated according to the following definitions for quality and impact:
+
+<!-- Using reStructuredText table here, otherwise the raw Markdown is greater than the 120-character line width -->
+```eval_rst
++-------+------------------------------------------------------+-------------------------------------------------------+
+| RAG   | Assumption quality                                   | Assumption impact                                     |
++=======+======================================================+=======================================================+
+| GREEN | Reliable assumption, well understood and/or          | Marginal assumptions; their changes have no or        |
+|       | documented; anything up to a validated & recent set  | limited impact on the outputs.                        |
+|       | of actual data.                                      |                                                       |
++-------+------------------------------------------------------+-------------------------------------------------------+
+| AMBER | Some evidence to support the assumption; may vary    | Assumptions with a relevant, even if not critical,    |
+|       | from a source with poor methodology to a good source | impact on the outputs.                                |
+|       | that is a few years old.                             |                                                       |
++-------+------------------------------------------------------+-------------------------------------------------------+
+| RED   | Little evidence to support the assumption; may vary  | Core assumptions of the analysis; the output would be |
+|       | from an opinion to a limited data source with poor   | drastically affected by their change.                 |
+|       | methodology.                                         |                                                       |
++-------+------------------------------------------------------+-------------------------------------------------------+
+```
+
+## Assumptions and caveats
+
+The log contains the following assumptions and caveats:
+
+<!-- Use reStructuredText contents directive to generate a local contents -->
+```eval_rst
+.. contents::
+    :local:
+    :depth: 1
+```
+
+### A1: 
```

Comment:
Stands for Assumption 1?

Reply:
Yes, I've made that more explicit, and added some missing placeholder text in commit [`c414e4a`](https://github.com/ukgovdatascience/govcookiecutter/pull/6/commits/c414e4add0dc6d1a0103cd984bdc198e07a849ab).

Marking as **Resolved**.